src/main/java/org/opensearch/knn/index/codec/KNN990Codec/NativeEngines990KnnVectorsReader.java

```diff
@@ -176,12 +189,28 @@ public void search(String field, byte[] target, KnnCollector knnCollector, Bits
      */
     @Override
     public void close() throws IOException {
+        // Clean up allocated vector indices resources from cache.
+        final NativeMemoryCacheManager nativeMemoryCacheManager = NativeMemoryCacheManager.getInstance();
+        // TMP
+        System.out.println(" &&&&&&&&&&&& [KDY] 990KV Cache manager BEFORE - " + nativeMemoryCacheManager.kdyPrintMap());
```

Comment:
Do we need print statement?

Reply:
This is an early review, I've not added testings yet.
This will be removed in the last commit :)